crates/nu-cli/src/eval_file.rs

```diff
@@ -98,6 +98,10 @@ pub fn evaluate_file(
         "CURRENT_FILE".to_string(),
         Value::string(file_path.to_string_lossy(), Span::unknown()),
     );
+    stack.add_env_var(
+        "ORIGINAL_CMDLINE".to_string(),
```

Comment:
Let's try process path if that's ok. We can always change it later if we don't like that either. I'll land it if you accept the changes.
```suggestion
        "PROCESS_PATH".to_string(),
```